README.md

```diff
@@ -1,40 +1,43 @@
+# Hello Visitor !! Welcome in PodCodar web App
 
-[![image](./public/images/vercel-logo.svg)][1]
+It's nice have you for here! This is PodCodar web a project of the community PodCodar with it's main objective is to teach good practices of development for our mentees.
 
-## Getting Started
+### Technologies
 
-First, run the development server:
+<p align="left">
+<a href="https://www.typescriptlang.org/" target="_blank" rel="noreferrer"> <img src="https://raw.githubusercontent.com/devicons/devicon/master/icons/typescript/typescript-original.svg" alt="typescript" width="40" height="40" paddingLeft="5px"/>  </a>
+<a href="https://reactjs.org/" target="_blank" rel="noreferrer"> <img src="https://raw.githubusercontent.com/devicons/devicon/master/icons/react/react-original-wordmark.svg" alt="react" width="40" height="40"paddingLeft="5px" />  </a>
+<a href="https://git-scm.com/" target="_blank" rel="noreferrer"> <img src="https://www.vectorlogo.zone/logos/git-scm/git-scm-icon.svg" alt="git" width="40" height="40"paddingLeft="5px"/> </a>
+<a href="https://nextjs.org/" target="_blank" rel="noreferrer"> <img src="https://cdn.worldvectorlogo.com/logos/nextjs-2.svg" alt="nextjs" width="40" height="40"paddingLeft="5px"/> </a>
+<a href="https://nodejs.org" target="_blank" rel="noreferrer"> <img src="https://raw.githubusercontent.com/devicons/devicon/master/icons/nodejs/nodejs-original-wordmark.svg" alt="nodejs" width="40" height="40"paddingLeft="5px"/> </a>
+<a href="https://www.linux.org/" target="_blank" rel="noreferrer"> <img src="https://raw.githubusercontent.com/devicons/devicon/master/icons/linux/linux-original.svg" alt="linux" width="40" height="40"paddingLeft="5px"/> </a>
+  </p>
 
-```bash
-npm run dev
-# or
-yarn dev
-```
+### Partners
 
-Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
+[![image](./public/images/vercel-logo.svg)][1]
 
-You can start editing the page by modifying `pages/index.tsx`. The page auto-updates as you edit the file.
+[1]: https://vercel.com/?utm_source=podcodar&utm_campaign=oss
 
-**⚠️ WARNING**: will need a `.env.local` file on the project root folder. Ask a copy to someone of the current dev team. The file should be like the following template:
+### Start Project
 
-```shell
-# file: .env.local
+- Install dependencies using library yarn whit command `yarn`.
+- Create archive `.env.local` in root folder and fill in the following fields :
 
-FIREBASE_APP_ID="***"
-FIREBASE_API_KEY="***"
-FIREBASE_PROJECT_ID="***"
-FIREBASE_MESSAGING_ID="***"
-FIREBASE_MEASUREMENT_ID="***"
-```
+        - FIREBASE_APP_ID="***"
+        - FIREBASE_API_KEY="***"
+        - FIREBASE_PROJECT_ID="***"
+        - FIREBASE_MESSAGING_ID="***"
+        - FIREBASE_MEASUREMENT_ID="***"
 
-## Learn More
+  after request credentials for team member.
 
-To learn more about Next.js, take a look at the following resources:
+- now enter a command `next start` and your project will be running.
```

Comment:
yarn instead of this line^^

Reply:
Let's avoid using the `next` command directly. If we use it directly, we need to install it as a global package instead of a local one.